src/App.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import "./App.css";
+import Button from "./Components/Button/ButtonComponent";
+import { ButtonSize, ButtonType } from "./Components/Button/consts";
+
+function App() {
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    console.log("Button clicked!", event);
+  };
+  return (
+    <div className="App">
+      <h1>hello world</h1>
+      <h1>hello world</h1>
```

Comment:
now that you have storybook up and running, clean the <App /> comp from theses stuff.